webapp/src/components/backstage/playbook_runs/playbook_run/playbook_run.tsx

```diff
@@ -250,42 +250,36 @@ const ColumnContainer = styled.div`
 `;
 
 const Container = styled(ColumnContainer)`
-    flex: 1;
-`;
-
-const MainWrapper = styled.div<{isRHSOpen: boolean}>`
-    flex: 1;
-    display: flex;
-    flex-direction: column;
-    max-width: ${({isRHSOpen}) => (isRHSOpen ? 'calc(100% - 400px)' : '100%')};
+    display: grid;
+    grid-auto-flow: column;
+    grid-auto-columns: 2fr minmax(400px, 1fr);
 
     @media screen and (min-width: 1600px) {
-        max-width: ${({isRHSOpen}) => (isRHSOpen ? 'calc(100% - 500px)' : '100%')};
+        grid-auto-columns: 2.5fr 500px;
     }
 `;
```

Comment:
Same is needed here

```suggestion
    overflow-y: hidden;
`;
```
min-height: 0 would also work